.ci/e2e/setup-e2e.ps1

```diff
@@ -0,0 +1,25 @@
+Write-Host "Downloading Functions Core Tools...."
+Invoke-RestMethod -Uri 'https://functionsclibuilds.blob.core.windows.net/builds/2/latest/version.txt' -OutFile version.txt
+Write-Host "Using Functions Core Tools version: $(Get-Content -Raw version.txt)"
+Remove-Item version.txt
+
+$currDir = Get-Location
+$output = "$currDir\Azure.Functions.Cli"
+
+
+if (-not (Test-Path env:CORE_TOOLS_URL)) 
```

Comment:
This should be
``` 
if (-not $env:CORE_TOOLS_URL)
{
}
```

Reply:
We discussed offline, it's ok for now because it's `env:CORE_TOOLS_URL` vs `$env:CORE_TOOLS_URL`.